### ISSUE TITLE
How about a login to save great and awws

### ISSUE THREAD

**Issue by IchordeDionysos** [webcomponents/webcomponents.org#868]
[Origin page](https://www.webcomponents.org/introduction)

**Comment by samuelli** [comment]
Hi @IchordeDionysos,

Are you aware you can star elements?

![image](https://cloud.githubusercontent.com/assets/787668/23485414/2b4d2c70-fe90-11e6-9239-d072a7628311.png)

We could expand this feature to let you see the ones you have starred in the past.

**Comment by IchordeDionysos** [comment]
Yeah I knew :D And that's a really good starting point! But only staring the GitHub repo is not the best. 
Until I can see, as you suggested, already steared ones it's easier to save and find them in a Link Collection. 
But I'm looking forward to see stared elements on webcomponents.org :D

**Comment by samuelli** [comment]
Yep - it's quite a bit of work, but I'll see what we can do!

**Comment by btopro** [comment]
is there any way of pinging github for a user's starred repos and then meshing that w/ repos that are mapped to webcomponents.org? Might reduce the work involved.

**Comment by IchordeDionysos** [comment]
In the new Version (v4) of Github API, they are using the GraphQL API.
In the User Object, there is a Connection called starredRepositories, what does exactly what you suggested.
How hard or easy it is to map them to the webomponent.org repo? I don't know :)

**Comment by IchordeDionysos** [comment]
https://developer.github.com/v4/reference/object/user/
